benchmarks/Base64Bench.hs

```diff
@@ -67,5 +67,5 @@ main =
       c <- random 1000
       d <- random 10000
       e <- random 100000
-      f <- random 1000000
+      f <- random 3000000000
```

Comment:
This give a baseline GB/s performance. At this point, `base64` is roughly 3GB/s, with `base64-bytestring` around 3GB/(1.6s)